network/topics/scoring.go

```diff
@@ -147,7 +151,7 @@ func scoreInspector(logger *zap.Logger, scoreIdx peers.ScoreIndex, logFrequency
 				zap.Float64("w6_ip_colocation_factor", w6),
 				zap.Float64("p7_behaviour_penalty", p7),
 				zap.Float64("w7_behaviour_penalty", w7),
-				zap.Any("invalid_messages", filtered),
+				zap.Strings("invalid_messages", invalidMessagesStats),
 			}
```

Comment:
A little bit above, we include a `p2_first_message_deliveries` field. This variable is a list of structures and it's also very big. It was introduced for debugging for scoring analysis some weeks ago, but I think now we can change it to be the sum through topics. I.e., the summation of `snapshot.FirstMessageDeliveries`*`topicScoreParams.FirstMessageDeliveriesWeight` through topics. If it's confusing, reach me so we can discuss it :)

Reply:
Nice. Lets do it also